sidebars.js

```diff
@@ -113,7 +113,15 @@ module.exports = {
           type: "category",
           label: "Technical Guide",
           collapsed: true,
+          
+          insight-page-update
+          items: [
+            "contributing/technical/setup-repo-with-git",
+            "contributing/technical/resolve-merge-conflicts",
+          ],
+      
```

Comment:
@Lymah123 there are a couple things to look closer here:

- On line 117: This is your branch name. Perhaps you accidentaly typed it here.
- Line 118 - 121 is duplicate with line 123.

These are the reasons whenever we run `npm start`, the docs won't built locally.

You can go to the `sidebar.js` in the `main` branch and copy-paste the whole lines to fix this.

Here how this object suppose to look like:

https://github.com/open-sauced/docs/blob/0cc70484d6e945ac77930e755bdde3196a2db9fe/sidebars.js#L112-L118

Reply:
@adiati98 , you can check now. Thank you!